### ISSUE TITLE
GA not working

### ISSUE THREAD

**Issue by xlc** [cennznet/cli#12]
Most likely sdk issue but could be just outdated dependency

```
$ cennz-cli.js repl --endpoint=wss://rimu.unfrastructure.io/public/ws
No wallet found. Pls run wallet:create to create wallet first.
failed to load wallet
api is loaded now
api utilApi genericAsset spotX is available in repl's context
(cennz-cli)> genericAsset.getFreeBalance
Thrown:
TypeError: Cannot read property 'freeBalance' of undefined
    at GenericAsset.get getFreeBalance [as getFreeBalance] (/Users/bryanchen/.config/yarn/global/node_modules/@cennznet/crml-generic-asset/GenericAsset.js:85:50)
(cennz-cli)> api.query.genericAsset.freeBalance(16000, '5GjvJYu1WJSWz4MN52wqtBkZmoJpXXcsLk6wypr6LrwfCQxG')
Promise { <pending> }
(cennz-cli)> 2019-06-05 17:47:53        RPC-CORE: subscribeStorage (keys: Vec<StorageKey>): StorageChangeSet:: Unable to determine type from 'Blake2_256'
```

both `genericAsset.getFreeBalance` and `api.query.genericAsset.freeBalance` are not working

**Comment by ianhe8x** [comment]
i can't reproduce it. Will leave this open and see if other ppl have same issue.

**Comment by ianhe8x** [comment]
just found what causes it, will fix in v0.7.1
